exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,10 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
```

Comment:
Should we update this to const? Or if you think it will be confusing for L9, open a ticket to do this in a couple of weeks?